apps/website/src/content/docs/notificationmarker.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: NotificationMarker
+description: Shows a notification marker dot at the top right of the component.
+thumbnail: #TODO
+---
+
+<p>{frontmatter.description}</p>
+
+<LiveExample src='NotificationMarker.main.jsx'>
+  <AllExamples.NotificationMarkerMainExample client:load />
+</LiveExample>
+
+## Usage
+
+Wrap the content with the `NotificationMarker` component.
+
+```jsx {2, 4}
+<IconButton label='Notifications' styleType='borderless'>
```

Comment:
Question (non-blocking): what is the intent/thinking behind showing `IconButton` in this static example, but not including it in the `main` example?

Reply:
In the `main` example, I was hoping to show the simplest use of the `NotificationMarker`.

In the static example, I was hoping to show the most likely usage, which is adding a NotificationMarker in the content of an `IconButton`.

Reply:
> the most likely usage

Should we add a few words to document it?

Reply:
> the most likely usage

Should we add a few words to document it?

Something like:
```diff
- Wrap the content with the `NotificationMarker` component.
+ The `NotificationMarker` component should be wrapped around the content which needs to have a marker ("dot") associated with it. This content is usually an icon, and is often used within an [`IconButton`](button#iconbutton).
```

Reply:
Sounds good, added (https://github.com/iTwin/iTwinUI/pull/1999/commits/6be8fb6c9332e5e98eff3fb8897a18aead3b5a77).